packages/common/src/compose-tunnel-agent.ts

```diff
@@ -18,3 +18,55 @@ export type ScriptInjection = {
   defer?: boolean
   async?: boolean
 }
+
+type Stringified<T> = {
+  [k in keyof T]: string
+}
+const parseScriptInjection = ({ pathRegex, defer, async, src }: Stringified<ScriptInjection>):
```

Comment:
Originally, it was in different files that were closed to the actual usage (not even residing in @common).
But after I removed the old labels, it seems that the module only contains configuration that is both specific to functionality (the type/params of ScriptInjection) and format (Compose labels) which makes it a config module with high awareness of both functionality and format.
For that reason, I've moved it there.
It might worth changing the name to config.ts, we can also split the compose labels and parsing/formatting to a different config.compose.ts, but I think it's premature.

Reply:
I've extracted the generic parsing logic for labels parsing/formatting to a compose-utils module.